src/ui/public/vislib/visualizations/_map.js

```diff
@@ -1,24 +1,28 @@
 import _ from 'lodash';
 import $ from 'jquery';
 import L from 'leaflet';
+import marked from 'marked';
+marked.setOptions({
+  gfm: true, // Github-flavored markdown
+  sanitize: true // Sanitize HTML tags
+});
+
 import VislibVisualizationsMarkerTypesScaledCirclesProvider from 'ui/vislib/visualizations/marker_types/scaled_circles';
 import VislibVisualizationsMarkerTypesShadedCirclesProvider from 'ui/vislib/visualizations/marker_types/shaded_circles';
 import VislibVisualizationsMarkerTypesGeohashGridProvider from 'ui/vislib/visualizations/marker_types/geohash_grid';
 import VislibVisualizationsMarkerTypesHeatmapProvider from 'ui/vislib/visualizations/marker_types/heatmap';
-export default function MapFactory(Private) {
+export default function MapFactory(Private, tilemap) {
 
   let defaultMapZoom = 2;
   let defaultMapCenter = [15, 5];
   let defaultMarkerType = 'Scaled Circle Markers';
 
+  let tilemapOptions = tilemap.options;
+  let attribution = marked(tilemapOptions.attribution);
```

Comment:
Alright, in that case I think we should keep it as is for now. We shouldn't change the existing API in this PR.